tsl/src/continuous_aggs/create.c

```diff
@@ -184,6 +185,7 @@ typedef struct CAggTimebucketInfo
 	Interval *interval;			  /* stores the interval, NULL if not specified */
 	const char *timezone;		  /* the name of the timezone, NULL if not specified */
 
+	FuncExpr *bucket_func;
```

Comment:
perhaps add a comment here for this field as well?

Reply:
done